src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -702,6 +702,12 @@ Additional information: {1}</value>
   <data name="Learn_more" xml:space="preserve">
     <value>Learn more</value>
   </data>
+  <data name="Build_plus_Live_Analysis_NuGet_Package" xml:space="preserve">
+    <value>Build + Live Analysis (NuGet Package)</value>
```

Comment:
In docs we use sentence casing, so this would be:
```suggestion
    <value>Build + live analysis (NuGet package)</value>
```
Infobar text might use a different capitalization scheme, however, in which case the existing capitalization might be fine.